apps/jobs-processing-service/src/.env.development

```diff
@@ -1,11 +1,31 @@
 NODE_ENV=dev
 PORT=3004
 
+# Novu
+MAX_NOVU_INTEGRATION_MAIL_REQUESTS=300
+# NOVU_EMAIL_INTEGRATION_API_KEY=
+
+# Storage Service
+# STORAGE_SERVICE=
```

Comment:
novu integration usecase will be moved to shared usecases lib, caching and storage could be moved to `application-generic`